core/utils/type-utils/src/main/java/datawave/data/normalizer/regex/NumericRegexEncoder.java

```diff
@@ -100,14 +98,9 @@ public class NumericRegexEncoder {
     private static final Logger log = Logger.getLogger(NumericRegexEncoder.class);
 
     /**
-     * Matches against any letter except for the letter d
-     */
-    private static final Pattern RESTRICTED_LETTERS_PATTERN = Pattern.compile(".*[\\p{Alpha}&&[^d]].*");
-
-    /**
-     * Matches against any non-ascii character
+     * Matches against any invalid character (anything besides reserved regex characters, numerics, and the letter d)
      */
-    private static final Pattern RESTRICTED_NONASCII_PATTERN = Pattern.compile(".*[^\\p{ASCII}].*");
+    private static final Pattern RESTRICTED_CHAR_PATTERN = Pattern.compile(".*[^.{,}()<>*+=!?^$|\\[\\]\\-\\\\0-9d].*");
```

Comment:
Maybe I'm parsing the regex wrong, but won't this match any number 0-9? Example "0" or "1"